README.md

```diff
@@ -12,15 +12,12 @@
   See the License for the specific language governing permissions and
   limitations under the License.
 -->
-[<img src="https://nifi.apache.org/assets/images/apache-nifi-logo.svg" width="300" height="126" alt="Apache NiFi"/>][nifi]
+|Apache NiFi                |  MarkLogic                |
+|:-------------------------:|:-------------------------:|
+|[<img src="https://nifi.apache.org/assets/images/apache-nifi-logo.svg" width="300" height="126" alt="Apache NiFi"/>][nifi]      |  [<img src="https://cdn1.marklogic.com/wp-content/uploads/2017/12/marklogic-logo-footer.svg" width="300" height="126" alt="MarkLogic"/>][marklogic]|
```

Comment:
I had no idea we had a CDN.

Reply:
I grabbed the link from the footer of marklogic.com.